common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -70,21 +70,16 @@
 <%def name="require_module(module_name, class_name)">
     <script type="text/javascript">
         (function (require) {
-            % if settings.REQUIRE_DEBUG:
-                require(['${module_name}'], function (${class_name}) {
-                    ${caller.body()}
-                });
-            % else:
-                ## The "raw" parameter is specified to avoid the URL from being further maninpulated by
-                ## static_replace calls (as woudl happen if require_module is used within courseware).
-                ## Without specifying "raw", a call to static_replace would result in the MD5 hash being
-                ## being appended more than once, causing the import to fail in production environments.
-                require(['${staticfiles_storage.url(module_name + ".js") + "?raw"}'], function () {
-                    require(['${module_name}'], function (${class_name}) {
-                        ${caller.body()}
-                    });
-                });
-            % endif
+            require(['${staticfiles_storage.url(module_name + ".js") + "?raw"}'], function (${class_name}) {
```

Comment:
I've implemented this locally, and I'm having issues getting it to work. For example, on the login page, after moving `/static/js/student_account/logistration_factory.js?raw` into a `<script>` tag, I get the following error on page load (which causes all the page content to not render):

```
require.js:165 Uncaught Error: Mismatched anonymous define() module: function ($, AccessView) {
            return function(options) {
                var $logistrationElement = $('#login-and-registration-container');

                new AccessView(_.extend(options, {el: $logistrationElement}));
            };
        }
http://requirejs.org/docs/errors.html#mismatch
```

It looks like this is because `logistration_factory.js` contains a `define()` call, which as the doc referenced in the error says, can't be used in code called from a `<script>` tag.

Ideas?